src/styles/styles.js

```diff
@@ -1520,8 +1519,14 @@ const styles = {
     },
 
     overlayStyles: (current) => ({
-        ...StyleSheet.absoluteFillObject,
-        backgroundColor: themeColors.overlay,
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/27353. The background color should have kept as overlay.